src/shared/utils/getIdKeyForNamespace.ts

```diff
@@ -24,6 +24,15 @@ import {
 
 import { Namespace } from '../types/namespaces';
 
+const fromSeparator = '|';
```

Comment:
What do you think about checking for the presence of the `fromSeparator` in the IDs and if it's there console error so we can catch it in sentry? I know it's unlikely for an ID to have this but just as an insurance